contracts/minter/src/contract.rs

```diff
@@ -522,13 +508,19 @@ pub fn mint_price(deps: Deps, admin_no_fee: bool) -> Result<Coin, StdError> {
     Ok(mint_price)
 }
 
+fn mint_count(deps: Deps, info: MessageInfo) -> Result<u32, StdError> {
+    let mint_count: u32 = (MINTER_ADDRS.key(info.sender).may_load(deps.storage)?).unwrap_or(0);
+    Ok(mint_count)
+}
+
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
         QueryMsg::Config {} => to_binary(&query_config(deps)?),
         QueryMsg::StartTime {} => to_binary(&query_start_time(deps)?),
         QueryMsg::MintableNumTokens {} => to_binary(&query_mintable_num_tokens(deps)?),
         QueryMsg::MintPrice {} => to_binary(&query_mint_price(deps)?),
+        QueryMsg::MinterAddresses {} => to_binary(&query_minter_addresses(deps)?),
```

Comment:
Would this ever need be queried? Isn't it more for internal accounting?

Reply:
Initially thinking creators might want to know distribution of addresses doing the minting. Now that I'm thinking about it, seems like something no one asked for and few people really want. Removing this bloat.